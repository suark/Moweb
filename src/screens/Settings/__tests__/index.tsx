import React from 'react';
import renderer from 'react-test-renderer';
import { Settings } from '..';

it('should render correctly', () => {
  const component = renderer.create(<Settings />);
  expect(component.toJSON()).toMatchSnapshot();
});
