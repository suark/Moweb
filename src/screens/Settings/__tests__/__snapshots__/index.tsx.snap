// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "rgb(28, 28, 30)",
        },
        undefined,
      ]
    }
  >
    Some Option
  </Text>
  <RCTSwitch
    accessibilityRole="switch"
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 31,
        "width": 51,
      }
    }
    value={true}
  />
  <Text
    style={
      Array [
        Object {
          "color": "rgb(28, 28, 30)",
        },
        undefined,
      ]
    }
  >
    Some Other Option
  </Text>
  <RCTSwitch
    accessibilityRole="switch"
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 31,
        "width": 51,
      }
    }
    value={false}
  />
  <Text
    style={
      Array [
        Object {
          "color": "rgb(28, 28, 30)",
        },
        undefined,
      ]
    }
  >
    Some Other Other Option
  </Text>
  <RCTSwitch
    accessibilityRole="switch"
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 31,
        "width": 51,
      }
    }
    value={false}
  />
</View>
`;
